dashboard/app/views/levels/_external_link.html.haml

```diff
@@ -5,3 +5,15 @@
   %h4= link_to @level.link_title, @level.url, target: '_blank'
 
   %div{style: 'padding-top: 100px'}!= t('external_links.disclaimer_md')
+
+  - if @script.try(:professional_learning_course?) && @script_level && @level.class == ExternalLink
+    -# Temporary hack - explicitly include the dialog helper so this submit button will post milestone
```

Comment:
Do we have a longer-term solution in mind? Since this is marked temporary, it would be nice to add a TODO: or just some more context on how it should be fixed.